lib/cron.js

```diff
@@ -1,4 +1,5 @@
 const cron = require('node-cron');
+const superagent = require('superagent');
```

Comment:
you don't need superagent as an extra dependency for it, we have request and request-promise-native already

Reply:
I know, Fabri added it before and I just kept it there. It will be replaced before it will be merged

Reply:
PR is still WIP